lms/static/certificates/sass/_utilities.scss

```diff
@@ -10,21 +10,163 @@
 // ------------------------------
 // #FUNCTIONS
 // ------------------------------
+@function rem($size) {
```

Comment:
Where is this function pulled from?

I think it's fine to add and use for now, but in future work we'll need to start removing these kinds of functions and align with bootstrap's methods of text sizing.

Reply:
@abutterworth this come from bourbon, and it's is used to convert px to rem and it's not only used in text size, but it's also used in padding, margin, size that is the reason I preferred to create the function while we defined how to handle these values.